client/landing/gutenboarding/stores/domain-suggestions/resolvers.ts

```diff
@@ -25,7 +25,7 @@ export function* __internalGetDomainSuggestions(
 	const suggestions = yield apiFetch( {
 		credentials: 'same-origin',
 		mode: 'cors',
-		url: addQueryArgs( url, queryObject ),
+		url: addQueryArgs( url, ( queryObject as unknown ) as InputArgsObject ),
```

Comment:
I have a question about this [here](https://github.com/Automattic/wp-calypso/pull/38219#issuecomment-565509806).

Reply:
Continued [there](https://github.com/Automattic/wp-calypso/pull/38219#issuecomment-566032411).